packages/GoogleAnalytics/src/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { InitializeOptions } from 'react-ga';
```

Comment:
Agreed with @moshthepitt, I don't think we should be exporting a reference to another library, I think if a user wants to reference this typing then they can install the react-ga types and reference it from there. 🤔